### PR TITLE
14202 creates json for relevant country data list

### PR DESCRIPTION
**Pull request by saricicekibrahim**

### CODE REVIEW

File: scripts/create_completeness_report.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    completeness.append(dataRecord)
+
+    fo = open(main_folder + "/report/completeness_" + country_code + ".json", 'w', encoding="utf-8")
+    fo.write(json.dumps(completeness))
+    fo.close()
+
+
+def main():
+    # make_profile folder
+    main_folder = sys.argv[1]
```
Comment (lgudyma-kontur) [inline scripts/create_completeness_report.py:145]: this variable is not needed, when this script is executed it is going to be run relative to main_folder, so we can use hardcoded directory path's

File: scripts/create_completeness_report.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    
+    # related country data folder
+    country_data_folder = sys.argv[2]
```
Comment (lgudyma-kontur) [inline scripts/create_completeness_report.py:148]: this var can be constructed based on **country_code**

File: scripts/create_completeness_report.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+    # country tag. "tza" for tanzania
+    country_code = sys.argv[3]
```
Comment (lgudyma-kontur) [inline scripts/create_completeness_report.py:151]: this is the only required variable

File: scripts/create_completeness_report.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+    files = get_datasets_filenames_datetime(country_code, country_data_folder)
+
+    dat = loadCsv(main_folder + "/static_data/completeness_test.csv", separator=',', skipheaders=True)
```
Comment (lgudyma-kontur) [inline scripts/create_completeness_report.py:155]: `    dat = loadCsv("/static_data/completeness_test.csv", separator=',', skipheaders=True)`

File: scripts/create_completeness_report.py [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    for idy, file in enumerate(files[0]):
+        multiFileNames.append(file)
+        multiDates.append(files[1][idy])
+
+    dataRecord["acquiredDatasets"] = multiFileNames
+    dataRecord["dateTime"] = multiDates
+
+    completeness.append(dataRecord)
+
+    fo = open(main_folder + "/report/completeness_" + country_code + ".json", 'w', encoding="utf-8")
```
Comment (lgudyma-kontur) [inline scripts/create_completeness_report.py:138]: main_folder is not needed
`    fo = open("/report/completeness_" + country_code + ".json", 'w', encoding="utf-8")`

File: scripts/create_completeness_report.py [hunk 6]
```diff
@@ -0,0 +1,160 @@
+import os, time
+import sys
+import json
+
+def loadCsv(strInputFile, encoding="utf-8", separator="|", skipheaders=False):
+    cells = []
+    if os.path.exists(strInputFile):
```
Comment (saricicekibrahim) [inline scripts/create_completeness_report.py:7]: no control here, if file exists

**Review by Zkir — APPROVED** [review]

**Pull request merged**